torch/nn/attention/flex_attention.py

```diff
@@ -846,7 +846,9 @@ def causal_mask(b, h, q_idx, kv_idx):
         Q_LEN = _round_up_to_multiple(Q_LEN, Q_BLOCK_SIZE)
     KV_LEN = _round_up_to_multiple(KV_LEN, KV_BLOCK_SIZE)
     if _compile:
-        inner_func = torch.compile(inner_func, fullgraph=True, dynamic=False)
+        inner_func = torch.compile(
+            torch.compiler.enable(inner_func), fullgraph=True, dynamic=False
```

Comment:
Err, don't you need to enable before compile?

Reply:
Oh, actually, I think it doesn't matter? compile and enable are commutative, aren't they...

Reply:
Only matters if we end up issuing a warning.